tests/transformers/searchservice/createDirectAnswer.ts

```diff
@@ -59,4 +61,42 @@ it('can create a FieldValueDirectAnswer', () => {
     fieldType: 'phone'
   };
   expect(actualDirectAnswer).toMatchObject(expectedDirectAnswer);
+});
+
+it('can create a custom FieldValueDirectAnswer', () => {
+  const apiFieldValueDirectAnswer = {
+    type: 'FIELD_VALUE',
+    answer: {
+      value: {
+        color: 'yellow',
+        os: 'android'
+      },
+      entityName: 'Obama Phone',
+      fieldName: 'Phone Description',
+      fieldApiName: 'phoneDescription',
+      fieldType: 'c010101.specialtype'
+    },
+    relatedItem: {
+      verticalConfigId: 'phone',
+      data: {}
+    }
+  };
+  const actualDirectAnswer = createDirectAnswer(apiFieldValueDirectAnswer);
+  const expectedDirectAnswer: CustomFieldValueDA = {
```

Comment:
I think we can sharpen the scope of this test if we use `expect.objectContaining` just to check that type is a DirectAnswerType.FieldValue and that `fieldType` is `'unknown'`

Reply:
good point!